apps/fulfillment-e2e/src/integration/fully-picking-a-picklist.cy.ts

```diff
@@ -18,9 +18,7 @@ describe('Fully pick a picking list', () => {
       });
 
       [pickingFragment.getProducts()].forEach((product) => {
-        const doneButton = product
-          .findByTestId('product-card-done')
-          .should('exist');
+        const doneButton = product.find('oryx-button').should('exist');
```

Comment:
Do we need the `.should('exist')` check here?

It seems like this can be replaced with `product.find('oryx-button').click({ multiple: true });` 

---

Also, `.find('oryx-button')` should be moved in page object / page fragment :)

Reply:
Yup, we need the `.should('exist')` because some product cards could be in edit mode and not with 'done button' visible